fetch_gazebo/CMakeLists.txt

```diff
@@ -49,6 +49,7 @@ install(
 
 install(
   PROGRAMS scripts/prepare_simulated_robot.py
+  PROGRAMS scripts/prepare_simulated_robot_pick_place.py
```

Comment:
This line fails with a CMake error on install. The second `PROGRAMS` keyword needs to be removed.

http://build.ros.org:8080/view/Idev/job/Idev__fetch_gazebo__ubuntu_trusty_amd64/2/console#console-section-0